server/types/requests/userRequest.ts

```diff
@@ -1,5 +1,5 @@
 export interface UserRequest {
 	username: string;
-	password: string;
+	password: string | undefined;
```

Comment:
the UserRequest interface may be created with a hashedPassword that could return undefined. TODO: figure out a better solution